### PR TITLE
Bugfix ecallisto client

### LINKED ISSUE #11: Add support for Phoenix-2 radio spectragrams
_From @ehsteve on August 10, 2012 13:27_

Add ability to download and load in Phoenix-2 radio spectragrams. Phoenix-2 data can be found at the following website http://soleil.i4ds.ch/solarradio/

_Copied from original issue: sunpy/sunpy#236_

### PR DESCRIPTION
**Pull request by samaloney**
### Description

Fixes #11  and the eCallisto client now returns all data the archive and removed the end time attribute see #60

### CODE REVIEW

**Comment by nabobalis** [comment]
What does the `e` stand for in `eCallisto`?

File: radiospectra/net/sources/ecallisto.py [hunk 1]
```diff
@@ -58,8 +57,14 @@ def pre_search_hook(cls, *args, **kwargs):
 
     def post_search_hook(self, exdict, matchdict):
         original = super().post_search_hook(exdict, matchdict)
-        # Files are 15 minute duration
-        original['End Time'] = original['Start Time'] + (15 * u.min - 1 * u.ms)
+        i0 = 0
+        if '_' in original['suffix']:
+            i0 = 1
+        original['ID'] = original['suffix'][i0:]
```
Comment (nabobalis) [inline radiospectra/net/sources/ecallisto.py:63]: We just cleaning up the ID by removing the front `_`?

File: radiospectra/net/sources/ecallisto.py [hunk 2]
```diff
@@ -29,20 +28,20 @@ class CALLISTOClient(GenericClient):
...
-    3 Results from the CALLISTOClient:
-           Start Time               End Time         Provider ... Observatory  ID
-    ----------------------- ----------------------- --------- ... ----------- ---
-    2019-10-05 23:00:00.000 2019-10-05 23:14:59.999 ECALLISTO ...      ALASKA  59
-    2019-10-05 23:15:00.000 2019-10-05 23:29:59.999 ECALLISTO ...      ALASKA  59
-    2019-10-05 23:30:00.000 2019-10-05 23:44:59.999 ECALLISTO ...      ALASKA  59
+    3 Results from the eCALLISTOClient:
+           Start Time        Provider Instrument Observatory  ID
+    ----------------------- --------- ---------- ----------- ---
+    2019-10-05 23:00:00.000 ECALLISTO  ECALLISTO      ALASKA  59
```
Comment (nabobalis) [inline radiospectra/net/sources/ecallisto.py:34]: The provider is also `ECALLISTO`?
I did not notice this.

File: radiospectra/spectrogram2/spectrogram.py [hunk 3]
```diff
@@ -673,6 +652,35 @@ def _read_fits(file):
             }
             return data, meta
 
+        # semi standard - spec in primary and time and freq in 1st extension
```
Comment (nabobalis) [inline radiospectra/spectrogram2/spectrogram.py:655]: How semi-standard?

**Comment by samaloney** [comment]
So essentially from the filename alone there's no way to tell about either the frequency or duration (see #60) so I think only returning only a start time makes sense and having no frequency information also makes sense.

Long term really need all those fits files to be indexed into some database and then search that but that's beyond the scope of radiospectra 😂

**Pull request merged**